src/server/cmd/job-shim/main.go

```diff
@@ -111,11 +111,13 @@ func do(appEnvObj interface{}) error {
 							// If someone received this signal, then they are
 							// responsible to exiting the program and release
 							// all resources.
+							fmt.Println("releasing resources...")
 							return
 						default:
 							// Otherwise, we just terminate the program.
 							// We use a non-zero exit code so k8s knows to create
 							// a new pod.
+							fmt.Println("terminating...")
```

Comment:
Did you mean to include these logging statements? If so could they be `lion` instead of `fmt`?